SR2020/2019-09-06-kickstart-date-announcement.md

```diff
@@ -0,0 +1,25 @@
+---
+to: Previously registered interested team leaders
+subject: SR2020 Kickstart Date Announcement
+---
+
+Hi,
+
+We're pleased to announce the dates and locations for the kickstart event for Student Robotics 2020.
+
+Kickstart will be on: XXXX
+
+This year we're running three Kickstart events:
+
+- The University of Southampton
```

Comment:
This is an organisation rather than a town, is that intentional?

Reply:
It is. They're 3 _locations_. I'm fully expecting to replace them with more specific locations once they're confirmed